splunk-otel-collector-chart/helm-charts/splunk-otel-collector/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
```

Comment:
This PR LGTM except for this file. This seems like a duplicate of helm-charts/splunk-otel-collector/Chart.yaml. Please remove.

Reply:
roger.